_posts/2023-02-13-homebrew-4.0.0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 4.0.0
+author: MikeMcQuaid
+redirect_from: /blog/4.0.0/
+---
+
+Today, I'd like to announce Homebrew 4.0.0. The most significant change since 3.6.0 enables significantly faster Homebrew-maintained tap updates by migrating from Git-cloned taps to JSON downloads.
+
+Major changes and deprecations since 3.6.0:
+
+- Using JSON files downloaded from [formulae.brew.sh](https://formulae.brew.sh) for package installation rather than local homebrew/core and homebrew/cask taps.
+  - **Please note: this is the largest change we have made to our update process since we split Homebrew/brew and Homebrew/homebrew-core repositories. Please bear with us, there may be a few bumps**.
+  - If you had previously set `HOMEBREW_NO_AUTO_UPDATE`, `HOMEBREW_NO_INSTALL_FROM_API` or `HOMEBREW_AUTO_UPDATE_SECS` to work around bugs or annoyances: please consider unsetting these and tweaking the values based on the new behaviour.
+  - Unless you are developing formulae or casks, you can `brew untap homebrew/core` and `brew untap homebrew/cask` to save some space.
+  - [`brew update` will now be run automatically less often (every 24 hours rather than every 5 minutes)](https://github.com/Homebrew/brew/pull/14376) and these auto-updates will be much faster as they no longer need to perform the slow `git fetch` of the huge homebrew/core and homebrew/cask taps' Git repositories.
+  - [This is now the default behaviour so the `HOMEBREW_INSTALL_FROM_API` variable has been removed and is a no-op](https://github.com/Homebrew/brew/pull/14412).
+  - If you wish to opt-out of this behaviour change, you can [`export HOMEBREW_NO_INSTALL_FROM_API=1`](https://github.com/Homebrew/brew/pull/14375). Please investigate [`HOMEBREW_API_AUTO_UPDATE_SECS`](https://github.com/Homebrew/brew/pull/14589) first.
```

Comment:
I suggest reversing the order of this sentence. You'll have a better chance of making the point you wanted to to people who stop reading. Will think of how to word it exactly.